centreon/www/install/php/Update-23.10.2.php

```diff
@@ -110,6 +110,25 @@
     );
 };
 
+$populateDahsboardTables = function(CentreonDb $pearDB): void {
+    if ($pearDB->isColumnExist('dashboard_widgets', 'name')) {
```

Comment:
This instruction is used to find out if a column exists before creating it.
This is not the case here.